tests/KubernetesClient.Tests/WatchTests.cs

```diff
@@ -772,11 +772,19 @@ public async Task WatchShouldCancelAfterRequested()
                 var cts = new CancellationTokenSource();
                 cts.CancelAfter(TimeSpan.FromSeconds(2));
 
-                await Assert.ThrowsAnyAsync<TaskCanceledException>(async () =>
+#if (NETSTANDARD2_0)
```

Comment:
As discussed, the test can be made independent of target framework. There should be a single assert for `OperationCanceledException`.

Reply:
Done.